src/components/ContextBar.js

```diff
@@ -58,6 +59,7 @@ export const ContextBar = ({ setDeletingWorkspace, setCloningWorkspace, setShari
   const getColorForStatus = status => Utils.cond(
     [_.upperCase(status) === 'RUNNING', () => colors.success()],
     [_.upperCase(status) === 'ERROR', () => colors.danger()],
+    [_.includes('ING', _.upperCase(status)), () => colors.accent()],
```

Comment:
I wonder if we should enumerate the possible status values here to be safe. We could have an array of them somewhere and still one-line it above and also on [this line](https://github.com/DataBiosphere/terra-ui/pull/2678/files#diff-4b313fc432ea524fbc7fdff4b0093404a4e65633ba99229f896fdfc15a0822c2L320).

Reply:
I think its safer to use 'ING', since this approach allows the backend to add further transition statuses without needing to remember this specific UI place (thinking of when we added things like 'PreCreating')

Reply:
@jdcanas @kyuksel 
With this change, "Provisioning" does not show in the orange dot and instead appears to be green (though perhaps I just can't tell the difference visually between colors.accent() and colors.success()). To me, this seems confusing because it appears that the app is ready to interact with.

Reply:
I think the issue is that the accent and success colors are too similar. Here is Galaxy "resuming" and Cromwell "running".

![image](https://user-images.githubusercontent.com/484484/142042639-cc9ea362-85b1-4b70-ac9c-0c0e3e5b30ff.png)